src/lib/Routes/Notify.ts

```diff
@@ -44,19 +52,17 @@ function notifyAPI(db: JsonDB.JsonDB, listeners: Array<wsWithHeartbeat>) {
 			});
 		}
 	
-		let notificationJson = JSON.stringify({
+		let notificationObject = {
```

Comment:
why isn't type specified here

Reply:
I fixed it in my fork but the new commit doesnt show up in the PR 🤔 
https://github.com/HappyDOGE/broadcaster/commit/539391fb69ed4db1a46a230fc308fa97581eaa60